Makefile

```diff
@@ -448,6 +449,10 @@ site/src/api/typesGenerated.ts: scripts/apitypings/main.go $(shell find codersdk
 	cd site
 	yarn run format:types
 
+docs/admin/prometheus.md: scripts/metricsdocgen/main.go
+	go run scripts/metricsdocgen/main.go
```

Comment:
should run prettier on the file immediately afterwards too to avoid fmt failures